projects/goharbor/harbor/Makefile

```diff
@@ -50,6 +50,27 @@ IMAGE_BUILD_ARGS=HARBOR_BASE_NAMESPACE
 FETCH_BINARIES_TARGETS=eksa/distribution/distribution
 FIX_LICENSES_XEIPUUV_TARGET=$(REPO)/vendor/github.com/xeipuuv/gojsonpointer/LICENSE.txt
 
+HAS_HELM_CHART=true
+
+HELM_GIT_TAG=v1.8.1
+HELM_SOURCE_REPOSITORY=harbor-helm
+HELM_DESTINATION_REPOSITORY=harbor/$(HELM_SOURCE_REPOSITORY)
+
+HELM_ADDITIONAL_KEY_VALUES=\
```

Comment:
Not for this PR, and i know this is a pattern we are following elsewhere, but I think we could probably introduce a helper here to make this cleaner.

Reply:
I have another PR where I use HELM_IMAGE_LIST instead of this. I want to get rid of this thing.